### PR TITLE
PLT-589: changed Admin to Team Admin in Manage Team members

### PR DESCRIPTION
**Pull request by stasvovk**

### CODE REVIEW

**Comment by coreyhulen** [comment]
I think there is 1 more place in the admin console this needs to change.  see components/admin_console/user_item.jsx

**Comment by stasvovk** [comment]
@coreyhulen that's done. thank you for the hint

**Comment by coreyhulen** [comment]
+1

File: web/react/components/admin_console/user_item.jsx [hunk 1]
```diff
@@ -113,6 +113,8 @@ export default class UserItem extends React.Component {
         if (user.roles.length > 0) {
             if (user.roles.indexOf('system_admin') > -1) {
                 currentRoles = 'System Admin';
+            } else if (user.roles === 'admin') {
```
Comment (coreyhulen) [inline web/react/components/admin_console/user_item.jsx]: Should be switched to use functions like Utils.isAdmin(roles), Utils.isSystemAdmin(roles), or Utils.isInRole(roles, inRole)

**Comment by crspeller** [comment]
+1

**Comment by jwilander** [comment]
+1

**Pull request merged**